python/cudf/cudf/core/index.py

```diff
@@ -1871,13 +1871,13 @@ def _get_dt_field(self, field):
     def is_boolean(self):
         return False
 
-    def ceil(self, field):
+    def ceil(self, freq):
```

Comment:
I'm in favor of changing this as I advocated for it in `round` below, but changing the parameter name here and for `floor` below (preexisting APIs) makes this PR a breaking change. @shwina do you see any issues? If not, we just need to change the tag.

Reply:
We could also argue that this is a bug (inconsistent with pandas) rather than a break.

Reply:
I’d say it’s a bug. Also with single-argument functions, keywords are much less common.

Reply:
I think the parameter change is fine and desirable. Agree that this is a bug, not a breaking change.